nucypher/policy/maps.py

```diff
@@ -351,4 +375,6 @@ def __bytes__(self):
             raise self.InvalidSignature(
                 "Can't cast a SignedTreasureMap to bytes until it has a blockchain signature "
                 "(otherwise, is it really a 'SignedTreasureMap'?")
```

Comment:
Interesting point. Is it really a ` SignedTreasureMap` object if it doesn't have a blockchain signature?